File: CursoPOOUber/Csharp/Paypall.cs
using PaymentTypes;

namespace PaymentTypes.Paypall 
{

    //Definition the "Paypall" class, which inherits from "Payment" class.
    public class Paypall : Payment 
    {

        //Attributes.
        public string Email { get; set; }


        public Paypall(int id, string email) : base(id)
        {
            this.Id = id;
            this.Email = email;
        }

        public void PrintPaypallData()
        {
            Console.WriteLine("This is the email of the paypall account of this user: " + Email);
        }
    }
}

File: CursoPOOUber/PHP/Paypall.php
<?php
require_once('Payment.php');
//Definition of the "Paypall" class, which inherits from "Payment" class.
class Paypall extends Payment {

    //Attributes
    public $email;

    public function __construct($id, $email){
        parent::__construct($id);
        $this->email = $email;
    }

    public function printPaypallData(){
        echo "This is the email of the paypall account of this user: " + $this->email;
    }
}
?>